### PR TITLE
Fix Small Bugs

### LINKED ISSUE #242: Make it so you can't transfer *from* location unless it is "Available"
And this really only applies to users right? Admins can do whatever they want...

### LINKED ISSUE #243: Make it so you can't transfer to inactive user

### LINKED ISSUE #277: Fix Search Input Overflow
When too many tags are added the search input doesn't handle it well at all

### LINKED ISSUE #278: Fix Purchased From Overflow on Add Tool Page

### PR DESCRIPTION
**Pull request by Jbcampbe**
Closes #242 
Closes #243 
Closes #277 
Closes #278

### CODE REVIEW

File: src/routes/tools.vue [hunk 1]
```diff
@@ -320,7 +321,7 @@ export default {
           user.label = `${user.first_name} ${user.last_name}`
         })
       }
-      return this.getAllUser || []
+      return this.getAllUser.filter(user => user.status === 'ACTIVE') || []
```
Comment (ekraihan) [inline src/routes/tools.vue]: I will be doing this filtering on the backend...

File: src/routes/tool-detail.vue [hunk 2]
```diff
@@ -635,9 +636,7 @@ export default {
         this.newModel = this.getTool.model_number
         this.newYear = this.getTool.year
         this.newPurchasedFrom = this.getTool.purchased_from
-        this.newPurchaseDate = new Date(
-          this.formattedDate(this.getTool.date_purchased)
-        )
+        this.newPurchaseDate = this.getTool.date_purchased && new Date(this.getTool.date_purchased)
```
Comment (ekraihan) [inline src/routes/tool-detail.vue]: this is a boolean? just checking

Comment (Jbcampbe) [reply src/routes/tool-detail.vue]: taking advantage of js :) if `date_purchased` is null then `d_p && new Date(d_p)` will just return null. If it is a truthy value then the date object will be returned

**Comment by Jbcampbe** [comment]
PR should be good to go.

**Pull request merged**